docs/planning/frontend/src/pages/EvaluatePage.jsx

```
@@ -0,0 +1,250 @@
+      {[1, 2, 3, 4, 5].map((star) => (
+        <button
+          key={star}
+          type="button"
+          className={`${sizeClasses} transition-colors ${
+            star <= (hover || value) ? 'text-yellow-400' : 'text-gray-300'
+          } hover:text-yellow-500`}
+          onMouseEnter={() => setHover(star)}
+          onMouseLeave={() => setHover(0)}
+          onClick={() => onChange(star)}
+        >
+          <svg fill="currentColor" viewBox="0 0 20 20">
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)